tests/robustness/multiclient_test/storagestats/storage_stats.go

```diff
@@ -0,0 +1,116 @@
+//go:build darwin || (linux && amd64)
+// +build darwin linux,amd64
+
+// Package storagestats contains logging mechanism
+// log disk space consumed by directories created by
+// robustness test framework before and after the test run.
+package storagestats
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	"github.com/kopia/kopia/tests/robustness/multiclient_test/framework"
+)
+
+var logFilePath string
+
+// DirDetails represents details about a directory,
+// path, and size.
+type DirDetails struct {
+	DirPath string `json:"dirPath"`
+	DirSize int64  `json:"dirSize"`
+}
+
+// LogStorageStats logs disk space usage of provided dir paths.
+func LogStorageStats(ctx context.Context, dirs []string) {
+	log.Printf("Logging storage stats")
```

Comment:
Logging statements can often be better handled outside of the package to ensure better separation of concerns. This allows the package to focus on its core functionality and leaves logging or/and other forms of handling (such as error handling, see my previous comment) to the calling code.

Reply:
done